### ISSUE TITLE
Would the model checkpoint be available in the future?

### ISSUE THREAD

**Issue by zwbx** [AlexanderKhazatsky/R2D2#26]
Hi, thanks for your great work. I would like know how can I get the model checkpoint, do I need to train it by myself?

**Comment by kpertsch** [comment]
Yes, we did not release the checkpoints since they are specific to the target domain data that they have been trained with. So to get a checkpoint you need to collect a small number of demos in your target domain and then you can use our code in the droid_policy_learning repository to train your own policy.
